cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -65,6 +65,9 @@ OPTIONS
        --debug
            Run tests in remote debugging mode, only when the '--skip-tests' flag is not used.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars.
```

Comment:
Wondering if we should just say

```suggestion
           Package the Choreo observability extension in the executables.
```

or

```suggestion
           Package the Choreo observability extension in the executable JAR file(s).
```

for consistency.

Reply:
+1 for the 2nd suggestion.

Reply:
Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5